mithril-explorer/aggregators-list.js

```diff
@@ -0,0 +1,7 @@
+export default [
+  "https://aggregator.release-preprod.api.mithril.network/aggregator",
+  "https://aggregator.pre-release-preview.api.mithril.network/aggregator",
+  "https://aggregator.testing-preview.api.mithril.network/aggregator",
+  "https://aggregator.api.mithril.network/aggregator",
```

Comment:
```suggestion

```
It will be decommissioned in the very next days, you can remove it from now

Reply:
done :ok_hand: